resources/views/community/components/event/aotw.blade.php

```diff
@@ -1,31 +1,49 @@
+@props([
+    'achievement' => [],
+    'game' => [],
+    'consoleName' => '',
+    'forumTopicId' => 0,
+])
+
 <?php
+$achievementId = $achievement->ID;
+$achievementName = $achievement->Title;
+$achievementDescription = $achievement->Description;
+$achievementPoints = $achievement->Points;
+$achievementRetroPoints = $achievement->TrueRatio;
+$achievementBadgeName = $achievement->BadgeName;
 
-use App\Site\Models\StaticData;
+$renderedGameTitle = renderGameTitle($game->Title);
+$achievementIconSrc = media_asset("/Badge/$achievementBadgeName.png");
+$gameSystemIconUrl = getSystemIconUrl($game->ConsoleID);
+?>
 
-$staticData = StaticData::first();
+<div class="component">
+    <h3>Achievement of the Week</h3>
 
-if ($staticData === null) {
-    return;
-}
+    <div class="bg-embed p-4 rounded border border-embed-highlight">
+        <a href="{{ route('achievement.show', $achievement->ID) }}" class="group flex gap-x-2 mb-3 text-text transition-transform hover:-translate-y-0.5">
+            <img src="{{ $achievementIconSrc }}" alt="Achievement of the week badge" width="64" height="64" class="w-16 h-16">
+            <div>
+                <p class="font-semibold leading-4 text-link group-hover:text-link-hover">{{ $achievement->Title }}</p>
+                <p class="text-xs mb-1">5 <span class="TrueRatio">(12)</span> Points</p>
+                <p class="text-xs">{{ $achievement->Description }}</p>
```

Comment:
I'm not convinced this behavior is desirable either. Yellow text is a link. Blue text is supplemental. I wouldn't expect to be able to open a new set by clicking on the Type data. Would it make sense to limit the hyperlink to just the Title?

Reply:
My (loosely held) opinion is the bigger hyperlink click/tap target is a better experience, however I will defer to your judgment. As of the latest commit, only the title and image are hyperlinked.

Reply:
Having the console text be linked, but not the achievement description seems inconsistent. I'd almost expect clicking on the console name to take me to a list of games for the console.

I don't use the site on mobile. Is it really that hard to click on the text? I'm going to defer judgment to @luchaos.

Reply:
I don't have that strong of an opinion on this.

My guts too tell me that with the current design the icon and title should be linked to the game while the console name should be linked to the console page.
Whereas if it should act as a button where everything is a label (we don't have those yet afaik) then it should look like a button, too? But maybe that's going too far as it would break the current look.

Reply:
Thinking ahead, avatars for our resources are in need of an overhaul in general. In this case I probably would've gone with a much smaller icon and inline game title + console variant as it's important but still just metadata for the achievement.

Reply:
>Thinking ahead, avatars for our resources are in need of an overhaul in general.

I agree. Mentioned here: https://github.com/RetroAchievements/RAWeb/pull/1658#discussion_r1258121986 This sort of design is dogfooding a potential alternative for game avatar UX.

It sounds like the path forward on this PR is:
* The console name should link to the console game list page.

Reply:
Feedback is now addressed in https://github.com/RetroAchievements/RAWeb/pull/1658/commits/33ccf23bc61440d968b9433974714096ef1277f8.

The console name now links to the system page. Additionally, points and retro points are no longer hardcoded 😵‍💫